dist/jquery.mark.js

```diff
@@ -89,19 +89,19 @@ function _classCallCheck(instance, Constructor) { if (!(instance instanceof Cons
         }, {
             key: "setupIgnoreJoinersRegExp",
             value: function setupIgnoreJoinersRegExp(str) {
-                return str.replace(/[^(|)]/g, function (val, indx, original) {
+                return str.replace(/[^(|)\\]/g, function (val, indx, original) {
                     var nextChar = original.charAt(indx + 1);
-                    if (/[(|)]/.test(nextChar) || nextChar === "") {
+                    if (/[(|)\\]/.test(nextChar) || nextChar === "") {
                         return val;
                     } else {
-                        return val + "\u0000";
+                        return val + "\0";
                     }
```

Comment:
The unit tests are using these files containing `\0`. As long as they are passing I don't think it's necessary. Even though I don't understand why an empty character `\u0000` is equivalent to `\0`